tests/test_utils.py

```diff
@@ -144,3 +146,64 @@ def test_nested_hash(self):
         nested_hash((1, 2, 3))
         nested_hash(set([1, 2, 3]))
         nested_hash({'foo': 'bar'})
+
+
+    def test_strip_removed_objects(self):
```

Comment:
I would break this test in separate ones that have meaningful names.

Reply:
actually that's a valid point, in general we should have used ``parametrize```

Reply:
Yes, that sounds even better!